src/v/kafka/server/group_manager.cc

```diff
@@ -249,21 +250,19 @@ ss::future<> group_manager::inject_noop(
   ss::lw_shared_ptr<cluster::partition> p,
   [[maybe_unused]] ss::lowres_clock::time_point timeout) {
     cluster::simple_batch_builder builder(
-      model::record_batch_type::raft_data, model::offset(0));
-    old::group_log_record_key key{
-      .record_type = old::group_log_record_key::type::noop,
-    };
-    builder.add_kv(std::move(key), iobuf());
+      model::record_batch_type::checkpoint, model::offset(0));
+
+    builder.add_kv(iobuf(), iobuf());
     auto batch = std::move(builder).build();
     auto reader = model::make_memory_record_batch_reader(std::move(batch));
```

Comment:
This reader is instantiated and never used, is this intended?

Reply:
good catch, removing it